src/adapters/adapter.js

```diff
@@ -57,7 +59,14 @@ export default class Adapter extends Configurable {
       this.bot.log.warning('Message received with undefined text.');
       return;
     }
-
+    if (this.prompts[user.id]) {
+      this.prompts[user.id].forEach((val, index) => {
+        if (val.cb(val.data,...arguments)) {
```

Comment:
is `...arguments` `val, index`, or the object passed into receiveWhisper? I'd expect it to be `val, index` here because you're using it within the forEach function.